src/extractors/tFunction.ts

```diff
@@ -54,20 +54,37 @@ function parseTCallOptions(
 
   // Try brutal evaluation first.
   const optsEvaluation = evaluateIfConfident(path);
-  if (optsEvaluation !== null && typeof optsEvaluation === 'object') {
-    res.contexts = 'context' in optsEvaluation;
-    res.hasCount = 'count' in optsEvaluation;
+  if (optsEvaluation !== null) {
+    if (typeof optsEvaluation === 'object') {
```

Comment:
This is 100% my fault, but I think we can just remove this case (`typeof optsEvaluation === 'object'`). It just adds useless complexity (in real world, objects would very rarely be evaluable) and the `path.isObjectExpression()` case below should be enough to catch all the cases we look for. I don't think it would break any test case.